content/applications/productivity/mail_plugins/outlook.rst

```diff
@@ -75,6 +75,25 @@ Connect your database
       Only a limited amount of *Company Insights* (*Lead Enrichment*) requests are available as a
       trial. This feature requires :ref:`prepaid credits <mail_plugins/pricing>`.
 
+   .. warning::
+      If, after a short while, the panel is still empty. It is possible that your browser cookies
+      settings policy did prevent it to load.
+      Note that this policy does also change if you are connected in "Incognito" mode on your
```

Comment:
Small changes to improve sentence flow.

`Note that these settings also change if you are in "Incognito" mode on your`